### PR TITLE
Fix some docblocks

### PR DESCRIPTION
**Pull request by derrabus**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | N/A
| License       | MIT
| Doc PR        | N/A

Backports from #33154.

### CODE REVIEW

**Comment by derrabus** [comment]
@tigitz fabbot wants to remove an annotation that I'd like to keep. Is the related to the new CS Fixer rule?

**Comment by nicolas-grekas** [comment]
New fabbot rule, with not perfect behavior, see https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4505

File: src/Symfony/Component/Workflow/Registry.php [hunk 1]
```diff
@@ -63,6 +63,13 @@ public function get($subject, $workflowName = null)
         return $matched;
     }
 
+    /**
+     * @param SupportStrategyInterface $supportStrategy
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Workflow/Registry.php]: see #33198

Comment (derrabus) [reply src/Symfony/Component/Workflow/Registry.php]: The doc block is correct here. The interface was replaced by a new one in 4.1.

**Comment by nicolas-grekas** [comment]
Thank you @derrabus.

**Pull request merged**